libs/components/code-examples/src/lib/modules/forms/checkbox/basic/example.component.html

```diff
@@ -1,32 +1,30 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.